lib/user-profile-manager/user-profile-manager.js

```diff
@@ -44,16 +47,18 @@ UserAttributeManager.prototype.init = function (options) {
 			break;
 		}
 	}
-	
+
 	this.userProfilesServerUrl = options[USER_PROFILE_SERVER_URL] || vcapServiceCredentials[USER_PROFILE_SERVER_URL];
-	if (!this.userProfilesServerUrl) {
-		logger.error("Failed to initialize user-attribute-manager.");
-		logger.error("Ensure your node.js app is either bound to an App ID service instance or pass required parameter to the constructor ");
+	this.oauthServerUrl = options[OAUTH_SERVER_URL] || vcapServiceCredentials[OAUTH_SERVER_URL];
+
+	if (!this.userProfilesServerUrl || !this.oauthServerUrl) {
+		logger.error("Failed to initialize user-manager.");
+		logger.error("Ensure your node.js app is either bound to an App ID service instance or pass required parameters to the constructor ");
 	}
-	logger.info(USER_PROFILE_SERVER_URL, this.userProfilesServerUrl);
+	logger.info(USER_PROFILE_SERVER_URL, this.userProfilesServerUrl, OAUTH_SERVER_URL, this.oauthServerUrl);
```

Comment:
make 2 logs instead of one i.e:
logger.info(USER_PROFILE_SERVER_URL, this.userProfilesServerUrl);
logger.info(OAUTH_SERVER_URL, this.oauthServerUrl);